whatprovides

```diff
@@ -35,23 +37,34 @@ update_database() {
 		rm -f "${DB_PATH}.gz"
 	fi
 
+	if ! ${FORCE_UPDATE} && [ -e "${DB_PATH}" ]; then
+		NEWER=(--time-cond "${DB_PATH}")
+	else
+		NEWER=()
+	fi
+
 	if ! ${QUIET}; then
 		echo "[*] Downloading the new database..." >&2
 		echo >&2
 
 		curl --fail --retry 3 --retry-connrefused --retry-delay 1 --location \
-			--output "${DB_PATH}.gz" "${DB_UPDATES_URL}"
+			"${NEWER[@]}" --output "${DB_PATH}.gz" "${DB_UPDATES_URL}"
 
 		echo >&2
-		echo "[*] Installing..." >&2
 	else
 		curl --silent --fail --retry 3 --retry-connrefused --retry-delay 1 \
-			--location --output "${DB_PATH}.gz" "${DB_UPDATES_URL}"
+			--location "${NEWER[@]}" --output "${DB_PATH}.gz" "${DB_UPDATES_URL}"
 	fi
 
-	rm -f "${DB_PATH}"
```

Comment:
oh also i removed this line since it seems pointless to manually delete a file when the very next line rewrites it